app/globals.css

```diff
@@ -17,3 +17,7 @@
   outline: 2px solid #428dff !important;
```

Comment:
Please define your hardcoded colors throughout your application.

Reply:
@cdsalty Hi Christopher. I'm having a bit of an issue with using the tailwind classnames in some cases with Framer. For me a lot of the time certain attributes with framer such as 'whileHover' can only accept a background color in the form of a style attribute. I've only been able to solved this by passing the hex value of the number in these cases. It's working but I agree it makes the code look inconsistent. So far I've been unable to find a workaround. The same goes in the globals.css file. I've been unable to use the tailwind classname instead of the hex value there.

Reply:
Let's look at this office hours next time. We can figure it out. Thanks for the update!